mlflow/gateway/client.py

```diff
@@ -337,3 +338,77 @@ def query(self, route: str, data: Dict[str, Any]):
                 raise MlflowException(message=timeout_message, error_code=BAD_REQUEST)
             else:
                 raise e
+
+    @experimental
+    def set_limits(self, route: str, limits: list[Dict[str, Any]] = []) -> LimitsConfig:
```

Comment:
Our CI runs in Python 3.8. This might help:
```suggestion
from typing import List

    def set_limits(self, route: str, limits: List[Dict[str, Any]] = []) -> LimitsConfig:
```